lib/migrator.js

```diff
@@ -2,21 +2,26 @@
 
 const TurndownService = require('turndown');
 const got = require('got');
-const { parse } = require('url');
-const { readFile } = require('hexo-fs');
+const parseUrl = require('url').parse;
```

Comment:
```suggestion
const { parse: parseUrl } = require('url');
```

It is easy to retain destructure with rename.